inductiva/_cli/cmd_user/costs.py

```diff
@@ -0,0 +1,93 @@
+"""List the user costs via CLI."""
+from typing import TextIO
+import argparse
+import sys
+
+from inductiva import users, _cli
+from inductiva.utils import format_utils
+
+
+def get_costs(args, fout: TextIO = sys.stdout):
+    """ Lists the user's costs.
+
+    Lists all costs for the user in the specified time period.
+    """
+    start_year = args.start_year
+    start_month = args.start_month
+    end_year = args.end_year
+    end_month = args.end_month
+    user_costs = users.get_costs(start_year=start_year,
+                                 start_month=start_month,
+                                 end_year=end_year,
+                                 end_month=end_month)
+
+    for element in user_costs:
+
+        compute_cost = float(element["components"]["compute"])
+        storage_cost = float(element["components"]["storage"])
+        total_cost = float(element["total"])
```

Comment:
I suggest using `decimal.Decimal` to avoid rounding issues. Something like this:

```python
import decimal

compute_cost = decimal.Decimal(str(element["components"]["compute"]))
```